pkg/lang/ir/python.go

```diff
@@ -166,6 +167,24 @@ func (g Graph) compilePyPIPackages(root llb.State) llb.State {
 			llb.Local(flag.FlagBuildContext), llb.Readonly)
 		root = run.Root()
 	}
+
```

Comment:
https://github.com/tensorchord/envd/pull/838/files#diff-e9007374abf264af40157329f8dd5adff0461ef389c98a8daa32eaf54b5db8adR119 also should be updated.